libs/api-client-tiger/src/gd-tiger-model/VisualizationObjectModelV1.ts

```diff
@@ -67,9 +67,9 @@ export namespace VisualizationObjectModelV1 {
      * @deprecated use {@link GrandTotal} instead
      */
     export interface ITotalItem {
-        measureIdentifier: LocalIdentifier;
+        measureIdentifier: AfmIdentifier;
```

Comment:
Should likely be `AfmLocalIdentifier`, right?

Reply:
Yep, updating in next commit 👍